addons/hr_work_entry_contract/static/src/components/work_entry_source_field/work_entry_source_field.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="hr_work_entry_contract.WorkEntrySourceField" t-inherit="web.RadioField">
+        <xpath expr="//input[hasclass('o_radio_input')]" position="attributes">
+            <attribute name="t-att-disabled">props.readonly or (item[0] === 'calendar' and isFullyFlexible)</attribute>
+        </xpath>
+
+        <xpath expr="//div[hasclass('o_radio_item')]" position="inside">
+            <span t-if="props.readonly or (item[0] === 'calendar' and item[0] === value and isFullyFlexible)"
```

Comment:
I am not sure why both ``item[0] === 'calendar'`` and ``item[0] === value`` are required here for the if to work :thinking:

Reply:
The goal is to display a warning only if the resource is fully flexible and the selected `value` (work entry source) is set to `calendar` (Working Schedule). Here `item` is iterating through the available work entry source (working schedule, planning, attendances), and without this check the warning will appear on each option of the radio.
![image](https://github.com/user-attachments/assets/cc9c5975-76aa-448b-83cf-d48ca3cb10ee)